assignments/blog/server/database/db.sql

```diff
@@ -2,12 +2,48 @@ CREATE DATABASE blog;
 
 \c blog
 
--- CREATE TABLE posts (
---     id SERIAL PRIMARY KEY,
---     title VARCHAR(250) NOT NULL,
---     author VARCHAR(150) NOT NULL,
---     content TEXT NOT NULL,
--- );
+CREATE TABLE posts (
```

Comment:
How does this script get called?

If it doesn't happen automatically, consider adding instructions here or somewhere else (like a Readme)

Should this script be idempotent? If not, how might we make sure "bad things" don't happen if it gets run twice (by accident) especially if this script is not run/managed automatically.

Do you have a plan for when the posts schema changes?  Say you want to add "posted_at" in the future, how might you do that without losing/dropping all your existing post data.

Reply:
1. planning on adding instructions in README for setting up database with db.sql 

2. The script is run manually. I didn't realize it could become an issue. What bad things do I need to think about and prevent from happening?

3. I have thought about adding another column. I believe sequelize migrations serves this purpose of controling changes to the database. I've seen it reading about sequelize, but didn't read about how to implement it.